### DISCUSSION TITLE
All/Selective Voice Chat.

### DISCUSSION THREAD

**Discussion by yujispin990** [winstxnhdw/lc-hax#128] [Ideas]
Make everyone or a selected individual hear me anywhere in the map. Hopefully its possible 😖

**Comment by winstxnhdw** [comment]
It’s probably possible but I won’t work on this soon. I am focusing on ESP.

**Reply by yujispin990** [reply]
Of course i understand, its great to know that its possible, i have some awesome uses for this, i really hope if it is not a burden and would love it if it gets added in the future 😊. Thank you again @winstxnhdw!

**Comment by yujispin990** [comment]
> It’s probably possible but I won’t work on this soon. I am focusing on ESP.
